### PR TITLE
Refactored styling of the frontpage to be better maintainable, starti…

### PR DESCRIPTION
**Pull request by royderks**
…ng with the highlights

### CODE REVIEW

File: src/app/App.jsx [hunk 1]
```diff
@@ -96,12 +98,34 @@ const App = ({
     siteId: MATOMO_CONFIG[getEnvironment(window.location.hostname)].SITE_ID,
   })
 
+  const StyledContainer = styled(Container)`
+    background-color: ${themeColor('tint', 'level1')};
+
+    position: relative;
+    @media screen and ${breakpoint('min-width', 'laptopM')} {
+      margin: 0 24px;
```
Comment (chrisvanmook) [inline src/app/App.jsx]: can use the themeSpacing util here 🎉

File: src/app/components/ContentContainer/ContentContainer.jsx [hunk 2]
```diff
@@ -5,13 +5,13 @@ import { breakpoint } from '@datapunt/asc-ui'
 
 const StyledContainer = styled.div`
   width: 100%;
-  margin-top: 20px;
-  margin-bottom: 20px;
+  margin-top: 16px;
```
Comment (chrisvanmook) [inline src/app/components/ContentContainer/ContentContainer.jsx]: same here

File: src/app/components/Homepage/AboutBlock.jsx [hunk 3]
```diff
@@ -2,14 +2,12 @@ import React from 'react'
...
 import aboutDataLinks from './services/about-data-links'
 import aboutSiteLinks from './services/about-site-links'
 import AboutLinkCard from './AboutLinkCard'
 
 /* istanbul ignore next */ const StyledAboutCard = styled(CardContainer)`
   ${({ showError }) => showError && ErrorBackgroundCSS}
-  ${blockTopMargin()}
 
   ${styles.ColumnStyle} {
     margin-bottom: 16px;
```
Comment (chrisvanmook) [inline src/app/components/Homepage/AboutBlock.jsx]: and here

File: src/app/components/Homepage/HighlightCard.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  &:focus {
+    background: none;
+
+    ${/* sc-selector */ styles.ImageCardStyle}::after {
+      ${focusImage()}
+    }
+  }
+`
+
+const HighlightCardHeading = ({ children, ...otherProps }) => (
```
Comment (chrisvanmook) [inline src/app/components/Homepage/HighlightCard.jsx]: do we need to seperate this? can't just render HighlightCardHeadingStyle inside the HighlightCard component

File: src/app/AppBody.jsx [hunk 5]
```diff
@@ -32,6 +34,36 @@ const EditorialOverviewPage = React.lazy(() => import('./pages/EditorialOverview
 const MapSplitPage = React.lazy(() => import('./pages/MapSplitPage'))
 const NotFound = React.lazy(() => import('./pages/NotFound'))
 
+// The Container from @datapunt/asc-ui isnt used here as the margins added do not match the ones in the design
+const Container = styled.div`
+  // Check if this should be moved to the @datapunt/asc-ui project
```
Comment (chrisvanmook) [inline src/app/AppBody.jsx]: I think so. Perhaps create issue in asc to create / refactor a backdrop component

File: src/app/components/EditorialPage/EditorialPage.jsx [hunk 6]
```diff
@@ -1,14 +1,23 @@
...
 import '../../../map/components/loading-indicator/LoadingIndicator.scss'
 import LoadingIndicator from '../../../shared/components/loading-indicator/LoadingIndicator'
-import Footer from '../Footer/Footer'
+
+const StyledContainer = styled(Container)`
+  min-height: 50vh; // Makes sure the loading indicator is displayed inside the page
+`
+const BodyStyle = styled.div`
+  background-color: ${themeColor('level', 'level1')}
+  min-height: calc(100% - 278px);
```
Comment (chrisvanmook) [inline src/app/components/EditorialPage/EditorialPage.jsx]: where does the 278px come from? If it's the header or footer's height, how does it respond to different viewports?

File: src/app/components/Header/Header.jsx [hunk 7]
```diff
@@ -11,7 +11,14 @@ import EmbedHeader from './EmbedHeader'
 import PrintHeader from './PrintHeader'
 
 const style = theme => css`
+  z-index: 9999;
+
   ${styles.HeaderNavigationStyle} {
+    @media screen and ${breakpoint('min-width', 'laptop')({ theme })} {
+      margin-left: 29px; // Check if this should be changed in @datapunt/asc-ui
```
Comment (chrisvanmook) [inline src/app/components/Header/Header.jsx]: Put this in an asc issue

File: src/app/components/Homepage/AboutCard.jsx [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+import React from 'react'
+
+const StyledCard = styled(Card)`
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+`
+
+const StyledCardContent = styled(CardContent)`
+  padding: ${themeSpacing(5)} ${themeSpacing(4)};
```
Comment (chrisvanmook) [inline src/app/components/Homepage/AboutCard.jsx]: can use shorthand here `${themeSpacing(5, 4)}`

File: src/app/components/Homepage/OrganizationBlock.jsx [hunk 9]
```diff
@@ -1,40 +1,58 @@
...
+import OrganizationCard from './OrganizationCard'
 import organizationLinks from './services/organization-links'
-import OrganizationLinkCard from './OrganizationLinkCard'
 
-const OrganizationBlockStyle = styled.div`
+const StyledCardContainer = styled(CardContainer)`
   background-color: ${themeColor('tint', 'level2')};
-  ${blockTopMargin(8)}
-  ${fullGridWidthContainer()}
+  padding: ${themeSpacing(8)} ${themeSpacing(4)};
```
Comment (chrisvanmook) [inline src/app/components/Homepage/OrganizationBlock.jsx]: shorthand

File: src/app/components/Homepage/OrganizationCard.jsx [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+import { Card, CardContent, Heading, Link, Paragraph, themeSpacing } from '@datapunt/asc-ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const StyledCard = styled(Card)`
+  border-top: 2px solid;
+  align-items: flex-start;
+  height: 100%;
+  width: 100%;
+  margin-bottom: 0px !important; // Check why the Card has a margin-bottom
```
Comment (chrisvanmook) [inline src/app/components/Homepage/OrganizationCard.jsx]: I think you should specify it using a double ampersand `&& { margin-bottom: 0; }` https://www.styled-components.com/docs/advanced#issues-with-specificity

File: src/app/components/Homepage/OrganizationCard.jsx [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+const StyledHeading = styled(Heading)`
+  margin: ${themeSpacing(3)} 0 ${themeSpacing(6)};
+`
+
+const StyledParagraph = styled(Paragraph)`
+  height: 100%;
+
+  // check where and WHY the margin-bottom rule is overriden in @datapunt/asc-ui
+  margin-bottom: ${themeSpacing(6)} !important;
```
Comment (chrisvanmook) [inline src/app/components/Homepage/OrganizationCard.jsx]: same as above

File: src/app/components/Homepage/SpecialCard.jsx [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+`
+
+const StyledCard = styled(Card)`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  padding: ${themeSpacing(2)} 0;
+  margin: ${themeSpacing(6)} 0;
+  @media screen and ${breakpoint('min-width', 'tabletM')} {
+    margin: ${themeSpacing(6)} ${themeSpacing(2)};
```
Comment (chrisvanmook) [inline src/app/components/Homepage/SpecialCard.jsx]: shorthand

File: src/app/AppBody.jsx (hunk 5 above)
Comment (royderks) [reply src/app/AppBody.jsx]: https://github.com/Amsterdam/amsterdam-styled-components/issues/133

File: src/app/components/EditorialPage/EditorialPage.jsx (hunk 6 above)
Comment (royderks) [reply src/app/components/EditorialPage/EditorialPage.jsx]: Deleted this as the `min-height` has no effect

File: src/app/components/Header/Header.jsx (hunk 7 above)
Comment (royderks) [reply src/app/components/Header/Header.jsx]: https://github.com/Amsterdam/amsterdam-styled-components/issues/165

File: src/app/components/EditorialPage/EditorialPage.jsx [hunk 13]
```diff
@@ -14,7 +14,6 @@ const StyledContainer = styled(Container)`
 `
 const BodyStyle = styled.div`
   background-color: ${themeColor('level', 'level1')}
-  min-height: calc(100% - 278px);
```
Comment (aditudorache) [inline src/app/components/EditorialPage/EditorialPage.jsx]: I thought this was needed to keep the publication page full screen with no space bellow the header. It didn't work as it with this setting, I have changed it to `min-height: calc(100vh - 330px);` in another branch (footer=280 + header=50)

Comment (royderks) [reply src/app/components/EditorialPage/EditorialPage.jsx]: Deleted this line and have no problems with rendering the page

Comment (aditudorache) [reply src/app/components/EditorialPage/EditorialPage.jsx]: Nice!

File: src/app/components/Header/Header.jsx [hunk 14]
```diff
@@ -11,7 +11,15 @@ import EmbedHeader from './EmbedHeader'
 import PrintHeader from './PrintHeader'
 
 const style = theme => css`
+  z-index: 9999;
```
Comment (chrisvanmook) [inline src/app/components/Header/Header.jsx]: 😮 ! lolwhut?

File: src/app/components/Header/Header.jsx [hunk 15]
```diff
@@ -30,6 +38,7 @@ const style = theme => css`
 
 const HeaderWrapper = styled.section`
   width: 100%;
+  z-index: 9999;
```
Comment (chrisvanmook) [inline src/app/components/Header/Header.jsx]: again? :)

**Pull request merged**